sdk/playwrighttesting/microsoft-playwright-testing/src/common/playwrightServiceConfig.ts

```diff
@@ -30,6 +31,10 @@ class PlaywrightServiceConfig {
       this.runId = options.runId;
       process.env[ServiceEnvironmentVariable.PLAYWRIGHT_SERVICE_RUN_ID] = this.runId;
     }
+    if (options?.runName) {
+      this.runName = options.runName;
```

Comment:
if options.runName is a dynamic field like new Date().toISOString(), then this will be set again and again (although, it will not create multiple entries, but still the approach is wrong). We would need a logic similar to what we are doing for multiple runId fix.

Reply:
yes Ik, but I thought it won't matter, will update